internal/pkg/manifest/validate.go

```diff
@@ -2000,6 +2031,80 @@ type validateARMOpts struct {
 	SpotFrom *int
 }
 
+func validateHealthCheckPorts(opts validateHealthCheckPortsOpts) error {
+	if opts.alb != nil {
```

Comment:
seems like if we _didn't_ make alb/nlb pointers (i.e., `alb HTTP; nlb NetworkLoadBalancerConfiguration`) we could get rid of both of these `!= nil` conditions 🤔